src/charts/line/line-chart.directive.js

```diff
@@ -0,0 +1,203 @@
+       <div class="col-md-12">
+         <div pf-line-chart config="config" chart-data="data" chart-height="custChartHeight" show-x-axis="custShowXAxis" show-y-axis="custShowYAxis"></div>
+       </div>
+       <hr class="col-md-12">
+       <div class="col-md-12">
+         <div class="row">
+           <div class="col-md-6">
+             <form role="form"">
+               <div class="form-group">
+                 <label class="checkbox-inline">
+                   <input type="checkbox" ng-model="custShowXAxis">X Axis</input>
+                 </label>
+                 <label class="checkbox-inline">
+                   <input type="checkbox" ng-model="custShowYAxis">Y Axis</input>
+                 </label>
+               </div>
+             </form>
+           </div>
+           <div class="col-md-3">
+                 <button ng-click="addDataPoint()">Add Data Point</button>
+           </div>
+         </div>
+       </div>
+     </div>
+   </file>
+
+   <file name="script.js">
+     angular.module( 'patternfly.charts' ).controller( 'ChartCtrl', function( $scope ) {
+
+       $scope.config = {
+         chartId: 'exampleLine',
+         grid: {y: {show: false}},
+         point: {r: 1},
+         color: {pattern: ["#0033CF", "#00CF33"]}
+       };
+
+       var today = new Date();
+       var dates = ['dates'];
+       for (var d = 20 - 1; d >= 0; d--) {
+         dates.push(new Date(today.getTime() - (d * 24 * 60 * 60 * 1000)));
+       }
+
+       $scope.data = {
+         xData: dates,
+         yData0: ['Created', 12, 10,10, 62, 17, 10, 15, 13, 17, 10, 12, 10, 10, 12, 17, 16, 15, 13, 17, 10],
+         yData1: ['Deleted', 10, 17, 76,14, 10, 10, 10, 10, 10, 10, 10, 17, 17, 14, 10, 10, 10, 10, 10, 10]
+       };
+
+       $scope.custShowXAxis = false;
+       $scope.custShowYAxis = false;
+
+       $scope.addDataPoint = function () {
+         $scope.data.xData.push(new Date($scope.data.xData[$scope.data.xData.length - 1].getTime() + (24 * 60 * 60 * 1000)));
+         $scope.data.yData0.push(Math.round(Math.random() * 100));
+         $scope.data.yData1.push(Math.round(Math.random() * 100));
+       };
+     });
+   </file>
+ </example>
+ */
+angular.module('patternfly.charts').directive('pfLineChart', function (pfUtils) {
+  'use strict';
+  return {
+    restrict: 'A',
+    scope: {
+      config: '=',
+      chartData: '=',
+      showXAxis: '=?',
+      showYAxis: '=?'
+    },
+    replace: true,
+    templateUrl: 'charts/line/line-chart.html',
+    controller: ['$scope',
+      function ($scope) {
+
+        // Create an ID for the chart based on the chartId in the config if given
+        $scope.lineChartId = 'lineChart';
+        if ($scope.config.chartId) {
+          $scope.lineChartId = $scope.config.chartId + $scope.lineChartId;
+        }
+
+        /*
+         * Convert the config data to C3 Data
+         */
+        $scope.getLineData = function (chartData) {
+          var lineData  = {
+            type: 'area'
```

Comment:
Sorry I didn't catch this before. The directive is line chart but this is actually creating an area chart. It should be renamed if the type is area or change type here to 'line'.